common/sched_queue_test.go

```diff
@@ -0,0 +1,91 @@
+package common
+
+import (
+	"testing"
+
+	"time"
+
+	"github.com/benbjohnson/clock"
+	wt "github.com/weaveworks/weave/testing"
+)
+
+func TestSchedCallsBasic(t *testing.T) {
+	InitDefaultLogging(testing.Verbose())
+	Info.Println("TestSchedCallsBasic starting")
+
+	const testSecs = 1000
+	cnt := 0
+	clk := clock.NewMock()
+	schedQueue := NewSchedQueue(clk)
+	schedQueue.Start()
+	defer schedQueue.Stop()
+
+	c := func() time.Time {
+		cnt++
+		return clk.Now().Add(time.Duration(1) * time.Second)
```

Comment:
Well, I always use the `time.Duration(XXX) * time.YYYY` pattern, but I don't mind breaking the exception here. I'll change it...